README.md

```diff
@@ -1,123 +1,98 @@
 # Node-api
 
-## Template project for RESTful API:s
+_This is specifically written for the `coding-interview` branch. See the `master` branch for a general introduction of the repository._
 
-In an attempt to simplify the process of starting up new Node.js based projects, there exists two template projects to use as a foundation.
+## Instructions
 
-The two projects are [node-web][web], a web server with express, and [node-api][api], a RESTful API.
-The node-web project CAS as a mechanism for authorisation and authentication.
+Make sure that the [Prerequisites](#prerequisites) are met, follow the steps in [Setup](#setup), and try out the application in [Getting started](#getting-started). Finally, try to solve the assignments in [Assignments](#assignments).
 
-### Where can I find the template projects?
+- Spend 1–2 hours on the assigments. You don’t need to finish all of them.
+- Please use at least one commit per assigment.
+- Add comments where needed.
+- Update tests if necessary, and feel free to add addtional tests.
+- Share the repository with the KTH representative.
 
-- [https://github.com/KTH/node-api.git][api]
-- [https://github.com/KTH/node-web.git][web]
+If you run into problems, see [Troubleshooting](#troubleshooting). Contact the KTH representative if you have any questions.
 
-It's important that we try to make changes that affect the template projects in the template projects themselves to make sure that all other projects based on the templates get the good stuff.
+## Prerequisites
 
-### How do I use this template project for a project of my own?
+- [Git](https://git-scm.com/)
+- [Node.js](https://nodejs.org/) and [npm](https://docs.npmjs.com/cli/v9/commands/npm)
+- [Docker](https://www.docker.com/)
+- [GitHub account](https://github.com/join)
 
-1. Create a new repository on Gita or Github.
-2. Clone the node-api repository by using:
+## Setup
 
-   ```sh
-   git clone git@github.com:KTH/node-api.git NEW_REPOSITORY_NAME
-   ```
+- Clone this repository ([KTH/node-api](https://github.com/KTH/node-api)) or create a new repository with this repository as a template.
```

Comment:
This should be ”Create a new repository with this repository as a template.”

Reply:
Fixed: 57ba727